### PR TITLE
redirect supervisor conditionally off contact form

### PR DESCRIPTION
**Pull request by ericcf**
Redirection from the form will be covered by acceptance tests.

### CODE REVIEW

File: app/controllers/participants_controller.rb [hunk 1]
```diff
@@ -73,7 +81,9 @@ def after_update_path
                      " " + t("actioncontroller.successfully")
 
     if current_user.nurse?
```
Comment (michaelwehrley) [inline app/controllers/participants_controller.rb]: This is a style question, any reason you did't right it this way: `if current_user.nurse? || referrer == TASKS_PAGE`

**Comment by michaelwehrley** [comment]
Not sure how @eschlange feels, but it looks good to me. :smile:

File: app/controllers/participants_controller.rb (hunk 1 above)
Comment (ericcf) [inline app/controllers/participants_controller.rb]: No, worth a refactor though!

**Comment by eschlange** [comment]
Looks good to me as well.

**Comment by ericcf** [comment]
Thanks @michaelwehrley and @eschlange.

**Pull request merged**